src/cr/cube/cubepart.py

```diff
@@ -927,6 +922,15 @@ def smoothed_dimension_dict(self):
         # from the matrix later on.
         return self._columns_dimension._dimension_dict
 
+    @lazyproperty
+    def sum(self):
```

Comment:
This should really be `sums`, shouldn't it? All of the other measures are plural, and this is a measure, and in general it contains multiple values, doesn't it? Pretty sure this should be `sums`, and all the related measure bits below would be plural too, like `_Sums`, `_BaseCubeSums`, etc.